Makefile

```diff
@@ -118,12 +118,17 @@ run-examples: build ## run examples against CSB on localhost (run "make run" to
 test: latest-csb lint run-integration-tests ## run the tests
 
 .PHONY: run-integration-tests
-run-integration-tests: ./tools/sqlfailover/build/sqlfailover_*.zip latest-csb provider-tests ## run integration tests for this brokerpak
+run-integration-tests: latest-csb ## run integration tests for this brokerpak
```

Comment:
You are right @blgm. Finally, I followed your advice about the integration tests. I beg your pardon for the delay, but I have been stuck with the child process that generates the make file and the declaration and propagation of variables through them.

I have associated a label to be able to filter and not run tests that require infrastructure. Tell me if you think it would be better to run it in another way. Now the GitHub action runs the (unitary) tests associated with the two providers